specs/electra/beacon-chain.md

```diff
@@ -1022,8 +1023,9 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
 
     # [Modified in Electra]
     if not self.notify_new_payload(
+            execution_payload, 
             parent_beacon_block_root,
-            execution_requests):
+            execution_requests_list):
         return False
```

Comment:
I accidentally deleted `execution_payload` in another PR. Can fix here. See:

* https://github.com/ethereum/consensus-specs/pull/3967/files#r1792295767